packages/tests/package.json

```diff
@@ -45,6 +47,18 @@
       "dependencies": [
         "build"
       ]
```

Comment:
```suggestion
      "dependencies": [
        "test:dev",
        "test:prod"
      ]
```

(And remove "command", I couldn't put that in the suggestion)

Reply:
(this is optional, just a neater way to write the same thing)

Reply:
will do!

Reply:
done!

Reply:
Make sure to also remove the command, or else it will run everything twice.

Reply:
I still see the command